nvdaHelper/vbufBase/backend.cpp

```diff
@@ -284,10 +284,6 @@ VBufStorage_controlFieldNode_t* VBufBackend_t::reuseExistingNodeInRender(VBufSto
 		LOG_DEBUG(L"existing node has no parent. Not reusing.");
 		return nullptr;
 	}
-	if(existingParent->identifier!=parent->identifier) {

```

Comment:
There's a comment a few lines above this which should probably be removed:

> ```
>        // We only allow reusing nodes that share the same parent.
>        // I.e. we don't allow reusing a node that existed in an entirely different part of the tree.
>        // If we did, this could possibly cause corruption in the virtualBuffer as a node might move between two unrelated updates.
> ```